tests/tests_basic.py

```diff
@@ -12,3 +12,149 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import pytest
+import subprocess
+
+import os.path
+import hashlib
+
+###################
+# Setup path data #
+###################
+
+tests_path = os.path.dirname( os.path.abspath( __file__ ))
+samples_path = os.path.join( tests_path, 'samples/')
+base_path = os.path.abspath( os.path.join( tests_path, '../' ))
+scripts_path = os.path.join( base_path, 'scripts/')
+
+####################
```

Comment:
when you add more tests, may be good to refactor helper functions into separate file. I assume you may re-use the md5 hash against master hashfile technique for future tests.